### ISSUE TITLE
Unexpected behaviour with ElementTree.ParseError

### ISSUE THREAD

**Issue by pmezard** [tiran/defusedxml#63]
Hello,

No sure if it is a bug or feature, but this is certainly a regression in 0.7.0. To reproduce:
```
import defusedxml.ElementTree
import xml.etree.ElementTree

try:
    defusedxml.ElementTree.fromstring("")
    print("OK")
except xml.etree.ElementTree.ParseError as e:
    print("PARSEERROR", e.__class__)
except Exception as e:
    print("NOT PARSEERROR", e.__class__)
```
which outputs:
```
NOT PARSEERROR <class 'xml.etree.ElementTree.ParseError'>
```
Unsurprisingly, it can be bisected to:
```
commit 3a48453780793c3e98251b0139e9e89e310fb617 (HEAD, refs/bisect/bad)
Author: Christian Heimes <christian@python.org>
Date:   Tue Jan 12 16:37:17 2021 +0100

    Restore xml.etree.ElementTree after patch
    
    Restore ``ElementTree`` attribute of ``xml.etree`` module after patching
```

Is this a bug or should we do this in a different manner? Regardless, this is very confusing.

Thanks.

**Comment by tiran** [comment]
This is an unforeseen side-effect of module reloading code. The exception classes look to be the same, but they are actually different classes.

```
>>> import defusedxml.ElementTree
>>> import xml.etree.ElementTree
>>> defusedxml.ElementTree.ParseError is xml.etree.ElementTree.ParseError
False
>>> defusedxml.ElementTree.ParseError
<class 'xml.etree.ElementTree.ParseError'>
>>> id(defusedxml.ElementTree.ParseError)
94084011057936
>>> xml.etree.ElementTree.ParseError
<class 'xml.etree.ElementTree.ParseError'>
>>> id(xml.etree.ElementTree.ParseError)
94084011051104
```

Looking into a fix now...

**Comment by tiran** [comment]
Fixed in release 0.7.1.

Thanks for the report!

**Comment by pmezard** [comment]
Merci !
